presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/StaticMetastoreConfig.java

```diff
@@ -51,6 +52,19 @@ public StaticMetastoreConfig setMetastoreUris(String uris)
         return this;
     }
 
+    public boolean isMultipleMetastoreEnabled()
+    {
+        return isMultipleMetastoreEnabled;
+    }
+
+    @Config("hive.metastore.load-balancing.enabled")
```

Comment:
nit: `hive.metastore.load-balancing-enabled`, also move this getters setters below the `getMetastoreUsername`, `setMetastoreUsername` to have the in the same order as the fields